blog-pipeline/src/blog_runner/llm.py

```
@@ -79,17 +79,7 @@ def wrapper(*args, **kwargs):
-# ── Pipeline Config (single source of truth) ──
-import importlib.util as _ilu
-
-_repo_root = Path(__file__).resolve().parent.parent.parent
-_cfg_path = str(_repo_root / "skills" / "blog-workflow" / "config" / "config_loader.py")
-if not Path(_cfg_path).exists():
-    _cfg_path = str(_repo_root / "config" / "config_loader.py")
-_cfg_spec = _ilu.spec_from_file_location("pipeline_config_loader", _cfg_path)
-_cfg_mod = _ilu.module_from_spec(_cfg_spec)
-_cfg_spec.loader.exec_module(_cfg_mod)
-pipeline_config = _cfg_mod.pipeline_config
+from blog_runner.config import pipeline_config
```

Ruff (0.15.9)

[error] Module level import not at top of file

(E402)